### PR TITLE
Added 155mmE

### PR DESCRIPTION
**Pull request by MustaphaTR**
Added elite weapon for Destroyer.

### CODE REVIEW

File: ra2/rules/allied-naval.yaml [hunk 1]
```diff
@@ -66,10 +66,19 @@ dest:
 	Selectable:
 		Bounds: 80, 80, 0, -6
 		Voice: AlliedNavalVoice
-	Armament:
+	Armament@PRIMARY:
```
Comment (abcdefg30) [inline ra2/rules/allied-naval.yaml]: Lowercase `primary` please.

File: ra2/rules/allied-naval.yaml [hunk 2]
```diff
@@ -66,10 +66,19 @@ dest:
...
 		Weapon: 155mm
 		LocalOffset: 768,0,0 # Not true, copy-pasted from SUB.
 		FireDelay: 2
+		UpgradeTypes: eliteweapon
+		UpgradeMaxEnabledLevel: 0
+		UpgradeMaxAcceptedLevel: 1
+	Armament@ELITE:
```
Comment (abcdefg30) [inline ra2/rules/allied-naval.yaml]: Same here with `elite`.

**Comment by MustaphaTR** [comment]
Updated.

**Comment by abcdefg30** [comment]
Looks good to me. :+1:

**Pull request merged**